### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by ghost**
PROBABLE -> PROBEABLE
This is follow-up to #3069?

### CODE REVIEW

File: Marlin/example_configurations/delta/biv2.5/Configuration.h [hunk 1]
```diff
@@ -545,11 +545,11 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = true; // set to true to invert the lo
   #if ENABLED(AUTO_BED_LEVELING_GRID)
 
     // Set the rectangle in which to probe.
-    #define DELTA_PROBABLE_RADIUS (DELTA_PRINTABLE_RADIUS - 10)
-    #define LEFT_PROBE_BED_POSITION -DELTA_PROBABLE_RADIUS
-    #define RIGHT_PROBE_BED_POSITION DELTA_PROBABLE_RADIUS
-    #define FRONT_PROBE_BED_POSITION -DELTA_PROBABLE_RADIUS
-    #define BACK_PROBE_BED_POSITION DELTA_PROBABLE_RADIUS
+    #define DELTA_PROBEABLE_RADIUS (DELTA_PRINTABLE_RADIUS - 10)
+    #define LEFT_PROBE_BED_POSITION -DELTA_PROBEABLE_RADIUS
```
Comment (Blue-Marlin) [inline Marlin/example_configurations/delta/biv2.5/Configuration.h:549]: `#define DELTA_PROBEABLE_RADIUS (DELTA_PRINTABLE_RADIUS - sqrt(sq(X_PROBE_OFFSET_FROM_EXTRUDER) + sq(Y_PROBE_OFFSET_FROM_EXTRUDER)))`
should do a better job than these obscure constants.

Comment (Blue-Marlin) [inline Marlin/example_configurations/delta/biv2.5/Configuration.h:549]: The complete section could go to `Conditionals.h`

**Comment by ghost** [comment]
I can't decide at my own discretion, whether I follow the your order or not. 
Because the alteration of calculation formula change DELTA_PROBEABLE_RADIUS of Kossel Pro and Kossel XL.

Kossel Pro

``` cpp
DELTA_PRINTABLE_RADIUS 127
X_PROBE_OFFSET_FROM_EXTRUDER -23
Y_PROBE_OFFSET_FROM_EXTRUDER -6
```

before:

``` cpp
DELTA_PROBEABLE_RADIUS (127 - 10) = 117
```

after:

``` cpp
DELTA_PROBEABLE_RADIUS (127 - sqrt(sq(-23) + sq(-6))) = 103.23027135...
```

Kossel XL

``` cpp
DELTA_PRINTABLE_RADIUS 140.0
X_PROBE_OFFSET_FROM_EXTRUDER 0.0
Y_PROBE_OFFSET_FROM_EXTRUDER 0.0
```

before:

``` cpp
DELTA_PROBEABLE_RADIUS (140.0 - 10) = 130.0
```

after:

``` cpp
DELTA_PROBEABLE_RADIUS (140.0 - sqrt(sq(0.0) + sq(0.0))) = 140.0
```

**Comment by ghost** [comment]
I click the thumbs up button just now, but it's miss-click.

**Comment by Blue-Marlin** [comment]
That only means the **example** configurations are a bit off. Not all Kossel XL have a FSR what would explain a 0-offset. 
That also means, the clipping for the points to probe for DELTAs works. :-)
probable_radius

**Comment by thinkyhead** [comment]
I presume that the compiler is (in 2016) smart enough to pre-compute those values, instead of all the `sqrt` and `sq` items ending up as function calls?

**Comment by Blue-Marlin** [comment]
@thinkyhead 
No, the pre-processor can not calculate this. But that does not matter. Even if you'd have to wait 21 instead of 20 seconds for a G29. That is not the time critical part of Marlin. Functions? I hope sqrt() and sq() are called as a function here - a real call and not inline - to save some space.
The trick here is that DELTA_PROBEABLE_RADIUS is never calculated during the pre-compilation - only tested for existence.

**Comment by thinkyhead** [comment]
Did I say "preprocessor"? Of course, the _compiler_ does optimize many math functions down to a constant value, as long as all the inputs are constant. For example, this code works just fine in GCC:

``` cpp
const double x = sqrt(pow(12,2) + pow(19,2));
```

In the compiled code, `x` is just assigned a constant value. No function calls.

**Pull request merged**